### PR TITLE
[Doc] Add abstract and teaser

### PR DESCRIPTION
**Pull request by LeoXing1996**

### CODE REVIEW

File: configs/biggan/metafile.yml [hunk 1]
```diff
@@ -64,7 +63,7 @@ Models:
     Task: Conditional GANs
   Weights: https://download.openmmlab.com/mmgen/biggan/biggan-deep_imagenet1k_128x128_cvt_hugging-face_rgb_20210728_111659-099e96f9.pth
 - Config: https://github.com/open-mmlab/mmgeneration/blob/master/configs/_base_/models/biggan/biggan-deep_256x256_cvt_hugging-face_rgb.py
-  In Collection: BigGAN
+  In Collection: README
```
Comment (nbei) [inline configs/biggan/metafile.yml:66]: Such behavior is not correct. Please further check it.

**Comment by nbei** [comment]
Since the script for generating meta files has been modified, you may pull the latest master branch and have a further check.

File: configs/biggan/metafile.yml (hunk 1 above)
Comment (LeoXing1996) [reply configs/biggan/metafile.yml:66]: It seems that `In Collection` in all metafiles are `README`.

Comment (nbei) [reply configs/biggan/metafile.yml:66]: No! It shouldn't be like this. You can have a look at the master branch.

**Pull request merged**